### ISSUE TITLE
[Accessibility] Close button in popup fails color contrast

### ISSUE THREAD

**Issue by amberhinds** [ProgressPlanner/brand-assets#14]
* [x] I've read and understood the [contribution guidelines](https://github.com/Emilia-Capital/.github/blob/main/CONTRIBUTING.md).
* [ ] I've searched for any related issues and avoided creating a duplicate issue.

### Please give us a description of what happened

The close button in the brand assets dialog fails color contrast.

Current CSS:
```
#brand_assets_logo_popover {
    button.close {
        border: none;
        background-color: #fff;
        color: #ddd;
        font-size: 1em;
        font-weight: bold;
        position: absolute;
        right: 8px;
        top: 5px;
        padding: 5px;
    }
}
```
Contrast analysis: [1.35:1](https://webaim.org/resources/contrastchecker/?fcolor=DDDDDD&bcolor=FFFFFF)

<img width="1434" height="838" alt="screenshot showing contrast failure for button" src="https://github.com/user-attachments/assets/f63651d9-c372-4bee-b03c-96048ec78dce" />

#### Step-by-step reproduction instructions
See the code referenced above in your stylesheet or inspect the element on the front end of the website.

#### Expected results
UI components should have at least a 3:1 contrast ratio.

#### Actual results
Current contrast is 1.35:1.

### Screenshots, screen recording, code snippet
If possible, please provide a screenshot, a screen recording or a code snippet which demonstrates the bug.

### Technical info

* If relevant, which editor is affected (or editors): 
- [ ] Block Editor
- [ ] Gutenberg Editor
- [ ] Elementor Editor
- [ ] Classic Editor 
- [ ] Other: 

* Which browser is affected (or browsers): 
- [x] Chrome
- [x] Firefox
- [x] Safari
- [x] Other: 

#### Used versions
* Device you are using:
* Operating system:
* PHP version:
* WordPress version: 
* WordPress Theme:
* Plugin version: 
* Gutenberg plugin version: 
* Elementor plugin version: 
* Classic Editor plugin version: 
* Relevant plugins in case of a bug:
